src/pos.cpp

```diff
@@ -78,11 +78,11 @@ bool ContextualCheckProofOfStake(const CBlockHeader& blockHeader, const Consensu
     }
 
     {
-        AssertLockHeld(cs_main);
-        uint32_t const mintedBlocksMaxDiff = static_cast<uint64_t>(mnView->GetLastHeight()) > blockHeader.height ? mnView->GetLastHeight() - blockHeader.height : blockHeader.height - mnView->GetLastHeight();
+//        AssertLockHeld(cs_main);
+//        uint32_t const mintedBlocksMaxDiff = static_cast<uint64_t>(mnView->GetLastHeight()) > blockHeader.height ? mnView->GetLastHeight() - blockHeader.height : blockHeader.height - mnView->GetLastHeight();
         // minter exists and active at the height of the block - it was checked before
-        uint32_t const mintedBlocks = mnView->GetMasternode(masternodeID)->mintedBlocks;
-        uint32_t const mintedBlocksDiff = mintedBlocks > blockHeader.mintedBlocks ? mintedBlocks - blockHeader.mintedBlocks : blockHeader.mintedBlocks - mintedBlocks;
+//        uint32_t const mintedBlocks = mnView->GetMasternode(masternodeID)->mintedBlocks;
+//        uint32_t const mintedBlocksDiff = mintedBlocks > blockHeader.mintedBlocks ? mintedBlocks - blockHeader.mintedBlocks : blockHeader.mintedBlocks - mintedBlocks;
```

Comment:
Can just remove the unused code

Reply:
I want it too, but i should know what it's trying to achieve, should we need to fix / implement? @monstrobishi

Reply:
No commented code in production please. Either delete or uncomment

Reply:
Will leave a info comment, the idea, about me, is to prevent single minter to mint a huge amount of continuous blocks (it can be indication for possible reorg, take over the network)